index.html

```diff
@@ -0,0 +1,130 @@
+            <h5>Here's a time line of Dr. Borlaug's life:
+            </h5>
+            <ul>
+                <li>
+                    <p><strong>1914</strong> - Born in Cresco, Iowa</p>
+                </li>
+                <!-- <li>
+                    <p><strong>1923</strong> - went to LACAS School</p>
+                </li> -->
+                <li>
+                    <p><strong>1933</strong> - Leaves his family's farm to attend the University of Minnesota, thanks to
+                        a
+                        Depression era program known as the "National Youth Administration"</p>
+                </li>
+                <li>
+                    <p><strong>1935</strong> - Has to stop school and save up more money. Works in the Civilian
+                        Conservation
+                        Corps, helping starving Americans. "I saw how food changed them", he said. "All of this left
+                        scars on me."
+                    </p>
+                </li>
+                <li>
+                    <p><strong>1937</strong> - Finishes university and takes a job in the US Forestry Service</p>
+                </li>
+                <li>
+                    <p><strong>1938</strong> - Marries wife of 69 years Margret Gibson. Gets laid off due to budget
+                        cuts. Inspired
+                        by Elvin Charles Stakman, he returns to school study under Stakman, who teaches him about
+                        breeding
+                        pest-resistent plants.</p>
+                </li>
+                <li>
+                    <p><strong>1941</strong> - Tries to enroll in the military after the Pearl Harbor attack, but is
+                        rejected.
+                        Instead, the military asked his lab to work on waterproof glue, DDT to control malaria,
+                        disinfectants, and
+                        other applied science.</p>
+                </li>
+                <li>
+                    <p><strong>1942</strong> - Receives a Ph.D. in Genetics and Plant Pathology</p>
+                </li>
+                <li>
+                    <p><strong>1944</strong> - Rejects a 100% salary increase from Dupont, leaves behind his pregnant
+                        wife, and
+                        flies to Mexico to head a new plant pathology program. Over the next 16 years, his team breeds
+                        6,000
+                        different strains of disease resistent wheat - including different varieties for each major
+                        climate on
+                        Earth.</p>
+                </li>
+                <li>
+                    <p><strong>1945</strong> - Discovers a way to grown wheat twice each season, doubling wheat yields
+                    </p>
+                </li>
+                <li>
+                    <p>1953- crosses a short, sturdy dwarf breed of wheat with a high-yeidling
```

Comment:
Nice job pointing this out :+1:. However, as I have stated in a previous comment, I suggest you phrase your comments politely as suggestions instead of orders to sound more friendly as well as elaborate more on why you requested this change. You can refer to this [link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-communicate) on how you can communicate better when doing reviews.